test/export/test_export.py

```diff
@@ -737,6 +738,61 @@ def forward(self, x: torch.Tensor, y: torch.Tensor) -> torch.Tensor:
         self.assertEqual(gm(*args), m(*args))
         args = (torch.randn(15, 3, 256, 256), torch.ones(15, 32, 256, 256))
         self.assertEqual(gm(*args), m(*args))
+    
+    @testing.expectedFailureLegacyExportNonStrict  # Some small change due to unbacked values getting regenerated
```

Comment:
any plan to follow up here? curious how bad it is

Reply:
I think it is just u0 is replaced with u1 in strict but everything else is same, i can look into it more but doesn't look bad.